main.py

```diff
@@ -59,3 +108,210 @@ async def request_classification(request: Request):
             "classification_scores": json.dumps(classification_scores),
         },
     )
+
+@app.post("/upload-and-classify")
+async def request_classification_upload(request: Request):
+    """
+    This function requests the upload image page.
+    Parameters
+    ----------
+    request: Request
+        The request that the user sends to the server.
+
+    Returns
+    -------
+    templates.TemplateResponse
+        The page with uploaded classified image and his score.     
+    """
+    # Load the form data from the post request
+    form = ClassificationUploadForm(request)
+    await form.load_data()
+
+    if form.is_valid():
+        # Retrive image_bytes loaded and model id from the form
+        bytes_img = form.image_bytes
+        model_id = form.model_id
+
+        # Call classify_image with choosen fetch_image function.
+        # fetch_image_bytes allows calling classify_image from raw bytes instead of a file
+        classification_scores = classify_image(model_id=model_id, img_id=bytes_img, fetch_image=fetch_image_bytes)
+
+        # Save the results in a json file
+        with open("app/static/results.json", "w") as f:
```

Comment:
What happens if multiple users ask for a classification job? The file would be overwritten. Isn't it best to return the results directly? Or, if you reuse them later for other things, another way to do it would be to store the results with a unique name and clean up after the response is returned to the user (or after a while anyways).